src/DefaultBuilder/testassets/Directory.Build.props

```diff
@@ -1,7 +1,9 @@
 <Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
   <PropertyGroup>
-    <OutputInRepoRoot>true</OutputInRepoRoot>
+    <!-- Tests do not work on Helix or when bin/ directory is not in project directory due to undeclared dependency on test content. -->
+    <BaseOutputPath />
+    <OutputPath />
```

Comment:
To confirm: The referencing project will be moved as a unit into the Helix payload and only built on those agents? And, the same thing happens where `testassets/Directory.Build.props` files are needed?

Reply:
Depends on how we decide to solve https://github.com/aspnet/AspNetCore/issues/10268.